src/algebraic_geometry/gluing.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import algebraic_geometry.presheafed_space.gluing
+
+/-!
+# Gluing Schemes
+
+Given a family of gluing data of schemes, we may glue them together.
+
+The construction should be "sealed" and considered as a black box, while only using the API
+provided.
```

Comment:
Can you make more precise which parts are the "public API"? Is that precisely the bits that are listed in `## Main definitions/results` below? If so, please say that.

Reply:
Oh this is probably not applicable here. I just copied this from the other doc string.
Things are already sealed up quite nicely in the presheafed_space/gluing file.